pkg/api/orchestrators_test.go

```diff
@@ -57,8 +57,8 @@ func TestOrchestratorUpgradeInfo(t *testing.T) {
 	}
 	orch, e := GetOrchestratorVersionProfile(csOrch)
 	Expect(e).To(BeNil())
-	// 1.5.7, 1.5.8, 1.6.6, 1.6.9, 1.6.11,
-	Expect(len(orch.Upgrades)).To(Equal(5))
+	// 1.5.7, 1.5.8, 1.6.6, 1.6.9, 1.6.11, 1.6.12
```

Comment:
@dmitsh could you please take a look at this file to make sure I didn't break the expected behavior of these tests by adding k8s versions 1.6.12 and 1.7.10?

Reply:
Also @dmitsh do I need to update the files in examples/kk8s-upgrade? eg. Does v1.6.11.json become v1.6.12.json or do I add another file for v1.6.12? Or neither?